src/Orc.NuGetExplorer/Management/Projects/DestFolder.cs

```diff
@@ -18,8 +19,11 @@ public class DestFolder : IExtensibleProject
 
         public DestFolder(string destinationFolder, IDefaultNuGetFramework defaultFramework)
         {
+            ArgumentNullException.ThrowIfNull(destinationFolder);
+            ArgumentNullException.ThrowIfNull(defaultFramework);
+
 #if NETCORE
-            var targetFramework = defaultFramework.GetHighest().FirstOrDefault();
+            var targetFramework = defaultFramework.GetHighest().First();
```

Comment:
First() will result in very hard to debug / understand exceptions. In such case, I think it's better to do FirstOrDefault and throw if null was returned